api/v1/users/index.php

```diff
@@ -17,4 +17,5 @@
  *
  */
 
-return new \APP\API\v1\users\UserHandler();
+// return new \APP\API\v1\users\UserHandler();
```

Comment:
Probably missing the comment "FIXME#7698: will be removed once merged pkp/pkp-lib#7698".